experimental/dispatch_profiler/profile_all.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Sets up a venv suitable for running IREE Dispatch Profiler and executes
+# a suite of runs. This is invoked by a Github workflow and can be invoked
+# locally.
+#
+# Recommend getting default 'python' to be python 3. For example on Debian:
+#   sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+# Or launch with python=/some/path
+
+set -euo pipefail
+
+TD="$(cd $(dirname $0) && pwd)"
+
+PYTHON="${PYTHON:-python3}"
+
+DISPATCH_PROFILER_IREE_BIN_DIR=${1:-""}
+if [ -z ${DISPATCH_PROFILER_IREE_BIN_DIR} ]; then
+  DISPATCH_PROFILER_IREE_BIN_DIR_FLAG="" 
+else
+  DISPATCH_PROFILER_IREE_BIN_DIR_FLAG="--iree-bin-dir=${DISPATCH_PROFILER_IREE_BIN_DIR}"
+ 
+fi
+
+DISPATCH_PROFILER_OUTPUT_DIR="${2:-"dispatch_profiler_output"}"
+DISPATCH_PROFILER_GENERATED_DIR="."
+VENV_DIR="dispatch-profiler.venv"
```

Comment:
nit: Maybe give an option to change generated dir and venv?

```suggestion
DISPATCH_PROFILER_GENERATED_DIR="${DISPATCH_PROFILER_GENERATED_DIR:-.}"
VENV_DIR="${DISPATCH_PROFILER_VENV_DIR:-dispatch-profiler.venv}"
```